### PR TITLE
Update manylinux docker images

### LINKED ISSUE #2091: /arch/TriCore/TriCoreMapping.o: In function `getVectorMinNumElements': TriCoreMapping.c:(.text+0x440): multiple definition of `getVectorMinNumElements'
From https://github.com/capstone-engine/capstone/actions/runs/5597666737/jobs/10236327487

```
In file included from arch/TriCore/TriCoreMapping.c:12:0:
arch/TriCore/../../cs_simple_types.h:325:13: warning: no previous prototype for ‘isValid’ [-Wmissing-prototypes]
 inline bool isValid(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:332:13: warning: no previous prototype for ‘isFloatingPoint’ [-Wmissing-prototypes]
 inline bool isFloatingPoint(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:343:13: warning: no previous prototype for ‘isInteger’ [-Wmissing-prototypes]
 inline bool isInteger(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:358:13: warning: no previous prototype for ‘isScalarInteger’ [-Wmissing-prototypes]
 inline bool isScalarInteger(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:365:13: warning: no previous prototype for ‘isVector’ [-Wmissing-prototypes]
 inline bool isVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:373:13: warning: no previous prototype for ‘isScalableVector’ [-Wmissing-prototypes]
 inline bool isScalableVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:379:13: warning: no previous prototype for ‘isFixedLengthVector’ [-Wmissing-prototypes]
 inline bool isFixedLengthVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:386:13: warning: no previous prototype for ‘is16BitVector’ [-Wmissing-prototypes]
 inline bool is16BitVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:395:13: warning: no previous prototype for ‘is32BitVector’ [-Wmissing-prototypes]
 inline bool is32BitVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:407:13: warning: no previous prototype for ‘is64BitVector’ [-Wmissing-prototypes]
 inline bool is64BitVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:421:13: warning: no previous prototype for ‘is128BitVector’ [-Wmissing-prototypes]
 inline bool is128BitVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:436:13: warning: no previous prototype for ‘is256BitVector’ [-Wmissing-prototypes]
 inline bool is256BitVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:452:13: warning: no previous prototype for ‘is512BitVector’ [-Wmissing-prototypes]
 inline bool is512BitVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:468:13: warning: no previous prototype for ‘is1024BitVector’ [-Wmissing-prototypes]
 inline bool is1024BitVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:482:13: warning: no previous prototype for ‘is2048BitVector’ [-Wmissing-prototypes]
 inline bool is2048BitVector(cs_data_type SimpleTy)
             ^
arch/TriCore/../../cs_simple_types.h:495:21: warning: no previous prototype for ‘getVectorElementType’ [-Wmissing-prototypes]
 inline cs_data_type getVectorElementType(cs_data_type SimpleTy)
                     ^
arch/TriCore/../../cs_simple_types.h:687:17: warning: no previous prototype for ‘getVectorMinNumElements’ [-Wmissing-prototypes]
 inline unsigned getVectorMinNumElements(cs_data_type SimpleTy)
                 ^
  GEN     capstone.pc
  LINK    libcapstone.so.5
  AR      libcapstone.a
/opt/rh/devtoolset-2/root/usr/bin/ar: creating ./libcapstone.a
./arch/TriCore/TriCoreMapping.o: In function `isValid':
TriCoreMapping.c:(.text+0x0): multiple definition of `isValid'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x240): first defined here
./arch/TriCore/TriCoreMapping.o: In function `isFloatingPoint':
TriCoreMapping.c:(.text+0x10): multiple definition of `isFloatingPoint'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x250): first defined here
./arch/TriCore/TriCoreMapping.o: In function `isInteger':
TriCoreMapping.c:(.text+0x40): multiple definition of `isInteger'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x280): first defined here
./arch/TriCore/TriCoreMapping.o: In function `isScalarInteger':
TriCoreMapping.c:(.text+0x70): multiple definition of `isScalarInteger'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x2b0): first defined here
./arch/TriCore/TriCoreMapping.o: In function `isVector':
TriCoreMapping.c:(.text+0x80): multiple definition of `isVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x2c0): first defined here
./arch/TriCore/TriCoreMapping.o: In function `isScalableVector':
TriCoreMapping.c:(.text+0x90): multiple definition of `isScalableVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x2d0): first defined here
./arch/TriCore/TriCoreMapping.o: In function `isFixedLengthVector':
TriCoreMapping.c:(.text+0xa0): multiple definition of `isFixedLengthVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x2e0): first defined here
./arch/TriCore/TriCoreMapping.o: In function `is16BitVector':
TriCoreMapping.c:(.text+0xb0): multiple definition of `is16BitVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x2f0): first defined here
./arch/TriCore/TriCoreMapping.o: In function `is32BitVector':
TriCoreMapping.c:(.text+0xe0): multiple definition of `is32BitVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x[320](https://github.com/capstone-engine/capstone/actions/runs/5597666737/jobs/10236327487#step:7:321)): first defined here
./arch/TriCore/TriCoreMapping.o: In function `is64BitVector':
TriCoreMapping.c:(.text+0x120): multiple definition of `is64BitVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x[360](https://github.com/capstone-engine/capstone/actions/runs/5597666737/jobs/10236327487#step:7:361)): first defined here
./arch/TriCore/TriCoreMapping.o: In function `is128BitVector':
TriCoreMapping.c:(.text+0x180): multiple definition of `is128BitVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x3c0): first defined here
./arch/TriCore/TriCoreMapping.o: In function `is256BitVector':
TriCoreMapping.c:(.text+0x1e0): multiple definition of `is256BitVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x420): first defined here
./arch/TriCore/TriCoreMapping.o: In function `is512BitVector':
TriCoreMapping.c:(.text+0x250): multiple definition of `is512BitVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x490): first defined here
./arch/TriCore/TriCoreMapping.o: In function `is1024BitVector':
TriCoreMapping.c:(.text+0x2c0): multiple definition of `is1024BitVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x500): first defined here
./arch/TriCore/TriCoreMapping.o: In function `is2048BitVector':
TriCoreMapping.c:(.text+0x320): multiple definition of `is2048BitVector'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x560): first defined here
./arch/TriCore/TriCoreMapping.o: In function `getVectorElementType':
TriCoreMapping.c:(.text+0x[380](https://github.com/capstone-engine/capstone/actions/runs/5597666737/jobs/10236327487#step:7:381)): multiple definition of `getVectorElementType'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x5c0): first defined here
./arch/TriCore/TriCoreMapping.o: In function `getVectorMinNumElements':
TriCoreMapping.c:(.text+0x440): multiple definition of `getVectorMinNumElements'
./arch/ARM/ARMMapping.o:ARMMapping.c:(.text+0x680): first defined here
collect2: error: ld returned 1 exit status
make: *** [libcapstone.so.5] Error 1
/opt/python/cp36-cp36m/lib/python3.6/site-packages/setuptools/dist.py:484: UserWarning: Normalizing '5.0.0post1' to '5.0.0.post1'
  warnings.warn(tmpl.format(**locals()))
error: [Errno 2] No such file or directory: 'libcapstone.so.5'
Error: Process completed with exit code 1.
0s
0s
```
cc @Rot127

### PR DESCRIPTION
**Pull request by Rot127**
closes https://github.com/capstone-engine/capstone/issues/2091

### CODE REVIEW

File: bindings/python/build_wheel.sh [hunk 1]
```diff
@@ -2,14 +2,7 @@
 set -e -x
 
 cd bindings/python
-sudo rm /usr/bin/python && sudo ln -s /opt/python/cp27-cp27m/bin/python /usr/bin/python; python -V
```
Comment (kabeor) [inline bindings/python/build_wheel.sh:5]: I think we should leave python2 here, don't delete it.

**Comment by aquynh** [comment]
No, we want to keep supporting python2.

**Comment by Rot127** [comment]
`Python3.6` was the problem of https://github.com/capstone-engine/capstone/issues/2091
Also it is EOL since 1,5 years. As well as the docker images (`manylinux1`).

The CI will just break more and more if we do not remove deprecated stuff. But if people want to use Python2 they can still do.

**Comment by Rot127** [comment]
But besides that was Python2 never used in this workflow anyway. The binary was copied, but in the `build_wheel.sh` script everything was exclusively Python3 again.

**Comment by kabeor** [comment]
Merge this first, but can you point where cause `Python3.6` no longer support? As far as I know, CI test works well on 3.6. @Rot127

**Comment by Rot127** [comment]
@kabeor Python3.6 seemed to work well until recently (#2091) (it only failed with those old versions, but it works now with more modern Python3.11). I could not pin point, what specifically the problem was. Maybe just the out-dated dockercross image (`manylinux1`, [not maintained](https://github.com/dockcross/dockcross#summary-cross-compilers) since 2 years).

But the newer dockercross images [drop support](https://github.com/pypa/manylinux) (see table) for EOL CPython.

**Pull request merged**